x-pack/plugins/observability/public/pages/alerts/containers/alerts_table_t_grid/alerts_table_t_grid.tsx

```diff
@@ -171,35 +170,59 @@ function ObservabilityActions({
   }, []);
 
   const casePermissions = useGetUserCasesPermissions();
-  const event = useMemo(() => {
-    return {
-      data,
-      _id: eventId,
-      ecs: ecsData,
-    };
-  }, [data, eventId, ecsData]);
-
   const ruleId = alert.fields['kibana.alert.rule.uuid'] ?? null;
   const linkToRule = ruleId ? prepend(paths.management.ruleDetails(ruleId)) : null;
 
+  const caseAttachments: CaseAttachments = useMemo(() => {
```

Comment:
@academo As discussed I will create a separate ticket to move all these in a separate file once this PR is merged

Reply:
@mgiota of course I can help you move that code out when you start working on it. We can make it a custom hook.